src/test/java/duke/parser/DateParserTest.java

```diff
@@ -0,0 +1,42 @@
+package duke.parser;
+
+import duke.exception.InvalidDateTimeException;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+class DateParserTest {
```

Comment:
An alternative way to test the parser is to compare the parsed result with an actual instance of `Date` object. See the example below:
```Java
Calendar calendar = Calendar.getInstance();
calendar.set(Calendar.DAY_OF_MONTH, 28);
calendar.set(Calendar.MONTH, Calendar.AUGUST);
calendar.set(Calendar.YEAR, 2019);
calendar.set(Calendar.HOUR_OF_DAY, 23);
calendar.set(Calendar.MINUTE, 59);
calendar.set(Calendar.SECOND, 0);
calendar.set(Calendar.MILLISECOND, 0);
Date date = calendar.getTime();
assertEquals(date, DateUtil.parseDate("28/8/2019 2359"));
```